### PR TITLE
Reduce bleed between neighboring keys

### PR DESCRIPTION
**Pull request by kdarkhan**
Small change updates sPWM logic by adding a small delay between disabling of the current column and enabling the next column.
The diff can be seen from this video. The current version looks [like this](https://i.imgur.com/RBfRU0F.mp4)

With proposed changes in the PR it will look [like this](https://i.imgur.com/gSZQwPI.mp4)

### CODE REVIEW

File: main.c [hunk 1]
```diff
@@ -459,6 +457,11 @@ void mainCallback(GPTDriver *_driver) {
   if (ledEnabled) {
 
     palClearLine(ledColumns[currentCol]);
+    for (int i = 0; i < NUM_ROW * 4; i++) {
+      if (i % 4 != 3) {
+        palClearLine(ledRows[i]);
```
Comment (kdarkhan) [inline main.c:462]: What we do here is not really important. The important thing is delay introduced by this instructions.

Comment (Codetector1374) [reply main.c:462]: LOL. Maybe then just stuff "nop"s there?

**Review by Codetector1374 — APPROVED** [review]
lgtm

File: main.c (hunk 1 above)
Comment (kdarkhan) [reply main.c:462]: It is hard to get the timing right. This accomplishes what is needed, and was simpler to understand :)

**Pull request merged**